src/test_typing_extensions.py

```diff
@@ -4229,8 +4234,8 @@ def test_signature_on_37(self):
     @skipUnless(TYPING_3_9_0, "NamedTuple was a class on 3.8 and lower")
     def test_same_as_typing_NamedTuple_39_plus(self):
         self.assertEqual(
-            set(dir(NamedTuple)),
-            set(dir(typing.NamedTuple)) | {"__text_signature__"}
+            set(dir(NamedTuple)) - {"__text_signature__"},
+            set(dir(typing.NamedTuple))
```

Comment:
Interesting, what was the failure here?

Reply:
I didn't look too hard honestly, something about `__text_signature__` unexpectedly not being in one of the sets. I assumed it was about whether or not Argument Clinic had been applied to the tuple constructor or something.

Reply:
the change seems fine, I'm just surprised the test broke!